CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ Mapbox welcomes participation and contributions from everyone.
 
 ## Features ✨ and improvements 🏁
 * Deprecated gesture settings `increaseRotateThresholdWhenPinchingToZoom` property. ([1632](https://github.com/mapbox/mapbox-maps-android/pull/1632))
+* Add experimental line-edge-clip property. ([1679](https://github.com/mapbox/mapbox-maps-android/pull/1679))
+* Introduce `location-indicator-opacity` to location indicator layer that allows to control layer opacity. ([1679](https://github.com/mapbox/mapbox-maps-android/pull/1679))
+* Enable asynchronous tile uploader by default to improve animation performance. ([1679](https://github.com/mapbox/mapbox-maps-android/pull/1679))
+* Vector tiles without symbols are not hold for fade-out animation so that less amount of vector tiles are managed at a time. ([1679](https://github.com/mapbox/mapbox-maps-android/pull/1679))
+* Introduce maps Version API. ([1679](https://github.com/mapbox/mapbox-maps-android/pull/1679))
```

Comment:
I would also point how to use it if we think that deserves the changelog entry

Reply:
this is an internal API without doc, so I suppose we should remove it from changelog.